expo-module.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "react-native-reanimated",
+  "platforms": ["android"]
```

Comment:
This enables the library to be autolinked by our scripts — are you sure we want to do this? Since it may cause side effects, I'd recommend to remove it. I guess we will need to introduce another field to disable autolinking 🤔 Also `name` is unnecessary as far as I remember

Reply:
we use expo autolinking because we need the generated `ExpoModulesPackageList.java` to get the list of handlers: https://github.com/expo/expo/blob/de6c8d2f8535df5c8a9620440daa30b9dafb30b2/packages/expo/android/src/main/java/expo/modules/ReactNativeHostWrapper.kt#L21-L22

it looks like gradle can handle duplicated dependencies so expo autolinking can coexist with rn-cli autolinking. but i'm fine if you have other thought than it.